### DISCUSSION TITLE
docs/ninjutsu/special-up-ranking-ninja/game-i-like

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#21] [General]
# docs/ninjutsu/special-up-ranking-ninja/game-i-like

如果说挂机类游戏，放一个是挂机，那么直接放80个！就是让游戏给你打工，你就是资本家！

https://996.ninja/docs/ninjutsu/special-up-ranking-ninja/game-i-like

**Comment by 67haha** [comment]
有没有什么养成类的休闲游戏

**Reply by MingGH** [reply]
那必须是！！原，神界原罪2，非常养成，一周目就长达120小时的休闲冒险，ipad也能玩
